.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/reference"]
+	path = test/reference
+	url = ../pixel-test-reference
```

Comment:
Hmm, this is expected actuallY?

Reply:
Yes, it's a feature.  This way, pixel-test-reference clone uses the same access method as the parent cockit-machines clone.

Reply:
but does that imply some specific local folder setup for the clones repos?

Reply:
No, this is only about the URLs that the clones are retrieved from.  The local repositories layout is defined by the `path` entry (which is of course also relative).